pages/clustering/high-availability.mdx

```diff
@@ -601,77 +549,101 @@ services:
     image: "memgraph/memgraph"
     container_name: coord1
     volumes:
-      - ./license.cypher:/tmp/init/license.cypher:ro
-      - ./HA_register.cypher:/tmp/init/HA_register.cypher:ro
+      - ./license.cypherl:/tmp/init/license.cypherl:ro
```

Comment:
You changed ports but not in registration queries above.

Reply:
And change it in chapter Add Coordinator Instance

Reply:
Also change in Data instances chapter exporting ports and in Coordinator instances also exporting ports.